src/annotator/components/adder-toolbar.js

```diff
@@ -96,8 +94,8 @@ export default function AdderToolbar({
     // @ts-ignore - TS doesn't know about our custom element types.
     <hypothesis-adder-toolbar
       class={classnames('annotator-adder', {
-        'annotator-adder--arrow-down': arrowDirection === 'down',
-        'annotator-adder--arrow-up': arrowDirection === 'up',
+        'annotator-adder--down': arrowDirection === 'up',
```

Comment:
The menu itself is actually "downward" when the arrow points "up" and vice-versa